prog.go

```diff
@@ -228,7 +229,7 @@ func newProgramWithOptions(spec *ProgramSpec, opts ProgramOptions, btfs btfHandl
 
 	fd, err := bpfProgLoad(attr)
 	if err == nil {
-		return &Program{internal.CString(logBuf), fd, spec.Name, spec.Type}, nil
+		return &Program{internal.CString(logBuf), fd, spec.Name, "", spec.Type}, nil
```

Comment:
Not a comment on your code, but we should probably switch to field specifiers for initializing structs since there are now adjacent string fields. @lmb

Reply:
Leaving them out is nice because it leads to a compile error when extending the struct. I don't feel very strongly about this. Let's do it in a different PR if you think it's possible.